doc/tags.yml

```diff
@@ -208,7 +220,7 @@ mapping_topics:
                        "nrf7001", "nrf7000", "nrf9160", "thingy91", "nrf9161", "nrf9151", "nrf5340",
                        "thingy53", "nrf52840", "nrf52833", "nrf52832", "nrf52820", "nrf52811",
                        "nrf52810", "nrf52805", "nrf21540", "npm1100", "npm1300", "npm6001",
-                       "development-kits", "prototyping-platforms", "kconfig"]
+                       "development-kits", "prototyping-platforms", "kconfig", "dect-nr+"]
```

Comment:
```suggestion
                       "development-kits", "prototyping-platforms", "kconfig"]
```

Reply:
Added to the tag to the DECT specific sample.